src/views/concepts/conceptPage.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+import BlueLinkButton from "shared/buttons/BlueLinkButton";
+import ContentBox from "../../shared/contentBox/ContentBox";
+import Accordion from "../../shared/accordion/Accordion";
+import { appRoutes } from "constants.js";
+import Typography from "shared/typography/Typography";
+
+export default () => {
+  return (
+    <ContentBox>
+      <NavLink to={appRoutes.HOME}>
+        <BlueLinkButton chevDirection="left" text="Terug naar kaart" />
```

Comment:
BlueLinkButton is een gekke naam als het geen link of button is. Nu is het gewoon link text of iets.